src/coprocessor/dag/expr/ctx.rs

```diff
@@ -321,6 +321,14 @@ impl EvalContext {
         }
     }
 
+    pub fn handle_wrong_argum(&mut self, err: Error) -> Result<()> {
```

Comment:
```suggestion
    pub fn handle_wrong_arguments(&mut self, err: Error) -> Result<()> {
```

Or `handle_wrong_args`.

Reply:
done